cache/in-memory/src/lib.rs

```diff
@@ -913,6 +913,11 @@ impl UpdateCache for Event {
             GuildEmojisUpdate(v) => c.update(v),
             GuildStickersUpdate(v) => c.update(v),
             GuildIntegrationsUpdate(_) => {}
+            GuildScheduledEventCreate(_) => {}
```

Comment:
do we want to cache these?

Reply:
I do not see any reason of why we should cache those right now but yes we can add it. I think that should be done in a separate PR though?